templates/autodevops/components/hasura.ts

```diff
@@ -16,9 +18,30 @@ export default async (): Manifests => {
       ? getGithubRegistryImagePath({ name, project: project ?? appName })
       : getHarborImagePath({ name: `${appName}-hasura` });
 
+  if (hasura === "exposed") {
+    exposed = true;
+  } else if (typeof hasura !== "boolean") {
+    exposed = !!hasura?.exposed;
+    resources = hasura?.resources;
+  }
+
   const config = {
-    config: { ingress: hasura === "exposed" },
-    deployment: { image },
+    config: { ingress: exposed },
+    deployment: {
+      container: {
+        resources: resources ?? {
+          limits: {
+            cpu: "500m",
+            memory: "512Mi",
+          },
+          requests: {
+            cpu: "100m",
+            memory: "64Mi",
+          },
+        },
```

Comment:
```suggestion
        resources,
```

si resources n'est pas fourni, le composant hasura propose déjà des defaults, p-e pas la peine d'override ici ?

Reply:
Yes 👍🏻